ext/extralite/common.c

```diff
@@ -243,6 +243,8 @@ int stmt_iterate(sqlite3_stmt *stmt, sqlite3 *db) {
       return 0;
     case SQLITE_BUSY:
       rb_raise(cBusyError, "Database is busy");
+    case SQLITE_INTERRUPT:
+      rb_raise(cError, "Query was interrupted");
```

Comment:
I think it would be better to add a new `Extralite::InterruptError` exception class to better align with the sqlite3 API.